tests/codeExecutor.test.js

```diff
@@ -0,0 +1,64 @@
+const {codeExecutor} = require("../utils/codeExecutor")
+
+describe("Code executor should work properly", () => {
+
+    it("should return true, when code is ok", async () => {
+        const data = {
+            language: 'python',
+            code: "inp = input()\n" +
+                "print(inp)",
+            input: "\"xd\"",
+            expectedResult: "xd"
+        }
+
+        const result = await codeExecutor.execute(data)
+
+        expect(result.passed).toBe(true)
+        expect(result.err).toBeFalsy()
+    })
+
+    it("should return false, if test didn't pass", async () => {
+        const data = {
+            language: 'python',
+            code: "inp = input()\n" +
+                "print(inp)",
+            input: "\"xd\"",
+            expectedResult: "other xd"
+        }
+
+        const result = await codeExecutor.execute(data)
+
+        expect(result.passed).toBe(false)
+        expect(result.err).toBeFalsy()
+    })
+
+    it("should show compilation error, if there's one", async () => {
+        const data = {
+            language: 'python',
+            code: "inp = input()\n" +
```

Comment:
Sprawdziłbym czy w jsie jest coś takiego jak w pythonie """ """ i najwyżej to splitować po \n, bo jakby ktoś chciał dodać większy kod to łatwiej będzie się czytał ten testowy kod